docs/source/config/webui.rst

```diff
@@ -47,7 +47,7 @@ On a web client side, a file ``config.js`` in a root of the project which contai
 
 Multiple servers could be configured for user to pick from. To disconnect from the current server and return to login screen, pick 'Disconnect' from the drop down at the top right corner of the UI.
 
-On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`__ should also be properly configured. In st2.conf, ``allow_origin`` property of the [api] section should contain the Origin header browser sends with every request. For example, if you have deployed UI on its own server and accessing it using url `http://webui.example.com:8000`, your config should look like that:
+On an |st2| side, `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`__ should also be properly configured. In st2.conf, ``allow_origin`` property of the [api] section should contain the Origin header browser sends with every request. For example, if you have deployed UI on its own server and accessing it using url `http://webui.example.com`, your config should look like that:
```

Comment:
It's on my TODO for testing - we manually injected CORS headers in nginx, but since this nginx module for injecting headers on non successful responses (e.g. 401 on auth failure) is not available in standard RHEL repos we decided to remove that.

To clarify - this only applies to non successful responses / status codes, we still correctly inject headers on all non 400 and 500 responses in Nginx.